qutebrowser/config/configdata.yml

```diff
@@ -954,6 +954,25 @@ hints.scatter:
 
     Ignored for number hints.
 
+hints.selectors:
+  default:
+    all: []
+    links: []
+    images: []
+    urls: []
+    inputs: []
+  type:
+    name: Dict
+    fixed_keys: ['all', 'links', 'images', 'urls', 'inputs']
```

Comment:
I couldn't think of a way to use the `webelem.Group` enum's names here other than hardcoding it like this... Do you have any suggestions?